src/displayapp/screens/FirmwareValidation.cpp

```diff
@@ -15,34 +16,50 @@ namespace {
 }
 
 FirmwareValidation::FirmwareValidation(Pinetime::Controllers::FirmwareValidator& validator) : validator {validator} {
+  lv_obj_t* title = lv_label_create(lv_scr_act(), nullptr);
+  lv_label_set_text_static(title, "Firmware");
+  lv_label_set_align(title, LV_LABEL_ALIGN_CENTER);
+  lv_obj_align(title, lv_scr_act(), LV_ALIGN_IN_TOP_MID, 10, 15);
+
+  lv_obj_t* icon = lv_label_create(lv_scr_act(), nullptr);
+  lv_obj_set_style_local_text_color(icon, LV_LABEL_PART_MAIN, LV_STATE_DEFAULT, LV_COLOR_ORANGE);
+  lv_label_set_text_static(icon, Symbols::check);
+  lv_label_set_align(icon, LV_LABEL_ALIGN_CENTER);
+  lv_obj_align(icon, title, LV_ALIGN_OUT_LEFT_MID, -10, 0);
+
   labelVersion = lv_label_create(lv_scr_act(), nullptr);
+  lv_label_set_recolor(labelVersion, true);
   lv_label_set_text_fmt(labelVersion,
-                        "Version: %lu.%lu.%lu\n"
-                        "ShortRef: %s",
+                        "#808080 Version# %lu.%lu.%lu\n"
+                        "#808080 Short Ref# %s\n",
                         Version::Major(),
                         Version::Minor(),
                         Version::Patch(),
                         Version::GitCommitHash());
-  lv_obj_align(labelVersion, nullptr, LV_ALIGN_IN_TOP_LEFT, 0, 0);
+  lv_obj_align(labelVersion, nullptr, LV_ALIGN_CENTER, 0, -40);
+  lv_label_set_align(labelVersion, LV_LABEL_ALIGN_CENTER);
 
   labelIsValidated = lv_label_create(lv_scr_act(), nullptr);
-  lv_obj_align(labelIsValidated, labelVersion, LV_ALIGN_OUT_BOTTOM_LEFT, 0, 0);
+  lv_obj_align(labelIsValidated, nullptr, LV_ALIGN_CENTER, 0, 10);
+  lv_label_set_align(labelIsValidated, LV_LABEL_ALIGN_CENTER);
+  lv_obj_set_auto_realign(labelIsValidated, true);
   lv_label_set_recolor(labelIsValidated, true);
-  lv_label_set_long_mode(labelIsValidated, LV_LABEL_LONG_BREAK);
-  lv_obj_set_width(labelIsValidated, 240);
+  // lv_label_set_long_mode(labelIsValidated, LV_LABEL_LONG_BREAK);
```

Comment:
Sorry if this is still in progress then please ignore for now, but before the merge commented out code should likely be removed.